CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Support for PHP 7.1 will be removed in Emogrifier 6.0.
 
 ### Removed
+- Drop support for Symfony 4.3 and 5.0
+  ([#918](https://github.com/MyIntervals/emogrifier/pull/918),
```

Comment:
URL should contain `issues` not `pull` (if required at all).

Reply:
Ah, damn copy'n'paste. Thanks for spotting these issues!